lesson17/index.js

```diff
@@ -0,0 +1,171 @@
+const url = "https://myjson.dit.upm.es/api/bins/8b9d";
+const main = document.querySelector("main");
+const sliderDirections = ["previous", "next"];
+
+const controlSlide = (direction) => {
+  const currentSlide = document.querySelector('[data-hidden="false"]');
+  const showSlide = currentSlide[direction];
+  currentSlide.setAttribute("data-hidden","true");
+  showSlide.setAttribute("data-hidden","false");
+  controlBtnBehavior();
+  updateFractionNum();
+}
+
+const slideImgTemplate = (index, slide) => {
+  const div = document.createElement("div");
+  div.classList.add("sliderImg");
+  div.id = `slide-${index}`;
+  if(index === 0){
+    div.setAttribute("data-hidden", "false");
+  } else {
+    div.setAttribute("data-hidden", "true");
+  }
```

Comment:
```suggestion
index === 0
    ? div.setAttribute("data-hidden", "false")
    : div.setAttribute("data-hidden", "true");
```

It's not must, so you don't have to fix.
But we can write this part like this too. I would just like to introduce it.
(If you already know, please ignore it 🤣 )
[ Conditional (Ternary) Operator](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

Reply:
[7071ce3](https://github.com/hippopo-moo/js-handson-work/pull/19/commits/7071ce302f67c510ceb3eb56ca8568b9db12b8f0)

Thank you so much!
I've never been good at ternary operators, so I haven't actively used them.😂!
Good opportunity to use it, so I did!